src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -371,18 +375,22 @@ const TooltipContent = styled(({children, ...p}) => (
   margin: 0;
 `;
 
-const TooltipRow = styled('tr')`
-  padding: ${space(0.5)} ${space(1)};
-  display: block;
+const TooltipRow = styled('tr')``;
```

Comment:
Is this still needed?

Reply:
It's not needed but I thought maybe keeping the styled component names would be nice, but actually cleaner code would be nicer, I'll remove this